Client.py

```diff
@@ -21,14 +23,25 @@ def Network_message(self, data):
         connection.Send(data)
         connection.Pump()
 
+    def sendPlayerInfo(self):
+        connection.Send({"action": "playerInfo", "playerInfo":
+            {"id": self.player.playerId, "x": self.player.x, "y": self.player.y, "imagePath": self.player.getImagePath()}})
+
     def Network_playersInfo(self, data):
         playersInfo = data["playersInfo"]
-        print("Players info: ", playersInfo)
+        for playerInfoElement in playersInfo:
+            for player in self.playersArray:
+                if playerInfoElement["id"] == player.playerId:
+                    player.x = playerInfoElement["x"]
+                    player.y = playerInfoElement["y"]
+                    self.imagePathArray[str(player.playerId)] = playerInfoElement["imagePath"]
 
     def Network_playerInfo(self, data):
         info = data["playerInfo"]
         self.player = Player(info["x"], info["y"], info["id"], self.screen)
-        print("My info: ", "id: ", self.player.playerId, "x: ", self.player.x, "y: ", self.player.y)
+        self.imagePathArray[str(self.player.playerId)] = info["imagePath"]
```

Comment:
Im wondering, if we have the `playersArray`, why do we create another instance of `Player` here, instead of assigning it to `player` : `self.player = PlayersArray[info["id"]`?

Reply:
Creating another instance is more effective for me, because it is responsible only for player movement and sending player's imagePath. The playersArray is only for holding players info and drawing players.

Interesting is the fact that, when I did `self.player = PlayersArray[info["id"]` player movement was lagging.